core/src/kura.rs

```diff
@@ -412,11 +415,9 @@ impl<IO: DiskIO> BlockStore<IO> {
             return Ok(None);
```

Comment:
please update the  documentation of the function

Reply:
Ok